genienlp/data_utils/example.py

```diff
@@ -88,19 +89,23 @@ def __init__(
         self,
         example_id: str,
         context: str,
+        context_spans: List[int],
```

Comment:
Is there a way to avoid adding `*spans` everywhere in the code?

Reply:
This is the fast/ clean way. Otherwise, you have to do multiple tokenization encoding and decodings along the way to make sure 1) you know where the spans are in source language 2) sentences are translated without quotation 3) you know where to put them back in the translated sentences, which would be much slower.